src/transport/mod.rs

```diff
@@ -175,6 +175,23 @@ bitflags! {
     }
 }
 
+/// ISR status field. Reading this register resets it to 0 and causes the device
+/// to de-assert the interrupt.
+///
+/// Ref: 4.1.4.5 ISR status capability
+#[derive(Copy, Clone, Default, Eq, FromBytes, PartialEq)]
+pub struct InterruptStatus(u32);
+
+bitflags! {
+    impl InterruptStatus: u32 {
+        /// Indicates that a virtqueue buffer was used
+        const QUEUE_INTERRUPT = 1;
+
+        /// Indicates that a virtio PCI device changed its configuration state
```

Comment:
Is this actually specific to PCI devices, or does it also apply to VirtIO devices with other transports?

Reply:
Section 4.2.2 MMIO Device Register Layout also mentions a "configuration change notification" bit in the InterruptStatus register.